### ISSUE TITLE
FileInput - Placeholder text doesn't meet accessibility requirements

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet-theme-hpe#284]
This issue can be seen in any of the FileInput storybook stories viewed with the hpe theme
https://storybook.grommet.io/?path=/story/input-fileinput-confirm-remove--confirm-remove&globals=theme:hpe

<img width="1581" alt="Screen Shot 2022-10-13 at 5 16 10 PM" src="https://user-images.githubusercontent.com/54560994/195730118-315b2397-b6e3-455a-9f9e-732f868544b2.png">

**Comment by vavalos5** [comment]
Here are a few good links to help with finding a solution: 

https://www.w3.org/WAI/tutorials/forms/instructions/

https://www.deque.com/blog/accessible-forms-the-problem-with-placeholders/

https://www.davidmacd.com/blog/is-placeholder-accessible-label.html

https://www.digitala11y.com/anatomy-of-accessible-forms-placeholder-is-a-mirage/

**Comment by vavalos5** [comment]
I'm assigning this ticket to Beza, however, i'm unable to assign ticket since this is a grommet issue.

If placeholder needs to be accessible, we will have to change all of the placeholder text colors for our input fields. Regardless, it will be good if we document whether or not placeholder text needs to meet accessibility.

### Deliverables:

- [ ] Define the problem
- [ ] Create a problem statement
- [ ] Do research + document findings
- [ ] Let team know direction based off of research
- [ ] Make any necessary placeholder text color changes
- [ ] Review with Vicky + Kenny + Jessica
- [ ] Connect w/ Jessica or a devs to make changes in Grommet + HPE Theme
- [ ] Update colors in Figma
- [ ] Create any necessary tickets that could follow

**Comment by vavalos5** [comment]
Ticket #2567 is similar to this issue. https://github.com/grommet/hpe-design-system/issues/2567

**Comment by taysea** [comment]
We are using `text-weak` for placeholder text in inputs which does meet contrast requirements and WCAG requirements for placeholder. It feels like we just need to update the FileInput placeholder to use it? I believe it's just that this line should change to `text-weak`: https://github.com/grommet/grommet-theme-hpe/blob/master/src/js/themes/hpe.js#L786

**Comment by taysea** [comment]
I don't think any of the other steps are necessary.

**Comment by vavalos5** [comment]
I wonder when we made that change since all Figma designs, including the new refactored inputs have the `text-xweak` color applied. I'm ok with moving with the `text-weak` color.

**Comment by taysea** [comment]
Gotcha, we must've gotten out of alignment, but it was related to this work:
- PR: https://github.com/grommet/grommet-theme-hpe/pull/234
- Initial ticket when `text-xweak` was introduced: https://github.com/grommet/grommet-theme-hpe/pull/234

**Comment by vavalos5** [comment]
I'll take this on and make changes in Figma.

**Comment by bezAdane** [comment]
Good point @taysea I think after going over the issue and doing some reading on it, there are a clear documentation from WCAG 2.1 - 3.3.2 (level A) & 1.4.1 (level A).) AND NOW  we have figma plagins that can show when some text fails so we can  add a few screen with description( document) and update the text to "weak"

**Comment by bezAdane** [comment]
> I'll take this on and make changes in Figma.
> 
> Change all placeholder text in inputs in Figma from `text-xweak` to `text-weak` to meet accessibility.

@vavalos5 let me know if you want to connect? since there might be other places we might need to make the necessary  updates in figma
